backend/apps/ai/management/commands/ai_create_event_chunks.py

```
@@ -0,0 +1,168 @@
+            metadata_parts.append(f"Location: {event.suggested_location}")
+
+        if event.latitude and event.longitude:
+            metadata_parts.append(f"Coordinates: {event.latitude}, {event.longitude}")
+
+        if event.url:
+            metadata_parts.append(f"Event URL: {event.url}")
+
+        return (
+            DELIMITER.join(filter(None, prose_parts)),
+            DELIMITER.join(filter(None, metadata_parts)),
+        )
```

Ruff (0.11.9)

No newline at end of file

Add trailing newline

(W292)